website/ref/language.md

```diff
@@ -1399,7 +1399,11 @@ Syntax:
 fn <name> <lambda>
 ```
 
-Define a function with a given name. The function behaves in the same way to the
+Define a function with a given name. The function name must be a valid
+[variable name](#variable). So `a` is valid but `a.b` is not valid as a
```

Comment:
I find it clearer to only use "so" to introduce general conclusions, rather than examples. Stick to "for example" for examples.

Reply:
Works for me. Done.